src/jit/target.h

```diff
@@ -1090,7 +1090,8 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
   // On Unix a struct of size >=9 and <=16 bytes in size is returned in two return registers.
   // The return registers could be any two from the set { RAX, RDX, XMM0, XMM1 }.
   // STOP_FOR_GC helper preserves all the 4 possible return registers.
-  #define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_FLOATRET | RBM_INTRET | RBM_FLOATRET_1 | RBM_INTRET_1))
+  //#define RBM_STOP_FOR_GC_TRASH     (RBM_CALLEE_TRASH & ~(RBM_FLOATRET | RBM_INTRET | RBM_FLOATRET_1 | RBM_INTRET_1))
+  #define RBM_STOP_FOR_GC_TRASH       (RBM_CALLEE_TRASH & ~(RBM_INTRET_1))
```

Comment:
If the commented-out code isn't needed, maybe you should delete it? You should update the comment, though.